### DISCUSSION TITLE
Upcoming major release 3.0

### DISCUSSION THREAD

**Discussion by jamesdaniels** [FirebaseExtended/reactfire#309] [Roadmap]
Just figured I'd kick off a thread to discuss the upcoming 3.0 release which allows use of the new ReactFire in stable React (making suspense optional). We hate to break APIs but we think this is a worthwhile change.

`3.0.0-rc.0` has been cut to NPM. Feel free to sound off it you have any feedback!

**Comment by prescottprue** [comment]
Hey! Glad to see support for stable react and suspense being able to fit into the same API even with the change to the API

Wanted to call out one thing around tagging of releases:
Maybe I'm misunderstanding, but I would expect that npm would match the github releases section (i.e. `v2.0.4` would be `latest` until `v3.0.0` is out of pre-release phase). This could be done by updating the releases section or by using the `next` tag when releasing to NPM (looks like this might already be getting done? 🤔 )

Wasn't sure exactly which way was preferred to go since it seems like docs including the README have already been getting updated to the newer API. Might also be nice to move the publishing of the package into a Github Action workflow that will handle all of that publishing/tagging for us. Open to making PR(s), just wanted to clarify intentions first

**Comment by zackdotcomputer** [comment]
Hey - I was wondering what is holding up 3.0 being promoted to the latest release? Is there a reason one should treat 3.0 as an RC still or is it safe to use in production?

**Comment by jhuleatt** [comment]
3.0.0 has been released.
